Containerfile

```
@@ -0,0 +1,32 @@
+# Use the NODE_VERSION argument to specify the Node.js version
+ARG NODE_VERSION

```

```suggestion
ARG NODE_VERSION=20
```

I would suggest setting a default value here so we can also build it without the build argument if we want. What do you think?

---

Ah before i forget. I used node-20 since this is the latest LTS version